services/venice-router/src/main/java/com/linkedin/venice/router/api/path/VenicePath.java

```diff
@@ -54,7 +54,7 @@ public abstract class VenicePath implements ResourcePath<RouterKey> {
    * When retry happens, the set will contain the storage nodes, which haven't finished the original scattered requests,
    * which will be treated as slow storage nodes, and Router will try to avoid retry requests to the storage nodes from this set.
    */
-  private Set<String> slowStorageNodeSet = Utils.newConcurrentSet();
```

Comment:
Can we remove this function from `Utils` too? It would not have any more users of it

Reply:
Yes, definitely... that was my intent. Thanks for catching it!